README.md

```diff
@@ -197,94 +180,147 @@ coastal area.
+- [OpenFOAM](https://www.openfoam.com/)
+- [SWASH](https://swash.sourceforge.io/)
+- [xBeach](https://oss.deltares.nl/web/xbeach/)
+- [GROMACS](https://www.gromacs.org/)
 
-scenario = molecules.scenarios.MDWaterBox(temperature = 300, box_size = 2.3)
-```
+If you would like other simulators to be added, contact us at [simulations@inductiva.ai](mailto:simulations@inductiva.ai).
 
-The user can specify the temperature (in Kelvin) and the box size (length of one of the cube's edges, in nanometers). The numbers above correspond to the default values.
+### Example
 
-After the initialization, we are ready to simulate the system:
+Example of how to use the simulators:
 
 ```python
-output = scenario.simulate(simulation_time = 10, nsteps_minim = 5000)
-```
-output = scenario.simulate(simulation_time = 10,
-                           nsteps_minim = 5000)
+from inductiva import fluids
 
-The simulate method initializes a simulation using cloud resources. The user can set the duration of the simulation in nanoseconds and the number of steps for the energy minimization.
+simulator = fluids.simulators.DualSPHysics()
 
-Finally, we can visualize the simulation: 
-```python
-view = output.render_interactive()
-view
+output_dir = simulator.run(input_dir="FlowCylinder",
+                           sim_config_filename="CaseFlowCylinder_Re200_Def.xml",
+                           output_dir="Flow",
+                           device="gpu")
 ```
-This renders the trajectory in an interactive visualization that can be manipulated in a jupyter notebook: 
-<p align="center">
-  <img width="900" height="300" src="https://github.com/inductiva/inductiva/assets/114397668/6cc809ed-085b-4fc6-b1a6-358be03cd061">
-</p>
 
+The user must specify the input directory, the simulation configuration file, the output directory and the device to run the simulation on.
 
-### ProteinSolvation
+Find more examples of simulations in the [tutorials section](https://github.com/inductiva/inductiva/tree/main/demos).
 
-The ProteinSolvation scenario models the dynamics of a protein whose structure is described by a PDB file. The protein is placed in a cubic box filled with water. If the protein has a non-zero electric charge, charged ions are added to the solution to neutralize the system. First, the system undergoes an [energy minimization](https://manual.gromacs.org/current/reference-manual/algorithms/energy-minimization.html) process to eliminate any steric clashes or structural issues within the protein-water system. After this, the position of the atoms in this system is updated according to Newton's equation in discrete time steps. The force that acts upon the particles is computed using standard molecular force fields.
 
-#### Example
+## Manage Resources
+
+**Inductiva API** provides a simple way to manage the hardware resources used to run the simulations. Users can launch virtual machines, list the available machines and terminate them. This is a feature available only to admins.
```

Comment:
Remove "This is a feature available only to admins"

Reply:
This is what we talked to day, about saying at the moment this is only available to admins.